misc/build.func

```diff
@@ -228,64 +227,75 @@ write_config() {
   if whiptail --backtitle "Proxmox VE Helper Scripts" --defaultno --title "Write configfile" --yesno "Do you want to write the selections to a config file?" 10 60; then
     FILEPATH="/opt/community-scripts/${NSAPP}.conf"
     if [[ ! -f $FILEPATH ]]; then
-      cat <<EOF >"$FILEPATH"
+  cat <<EOF >"$FILEPATH"
 # ${NSAPP} Configuration File
 # Generated on $(date)
 
 CT_TYPE="${CT_TYPE}"
 DISK_SIZE="${DISK_SIZE}"
 CORE_COUNT="${CORE_COUNT}"
 RAM_SIZE="${RAM_SIZE}"
+VERBOSE="${VERBOSE}"
+PW="${PW##-password }"
+#CT_ID=$NEXTID
 HN="${HN}"
 BRG="${BRG}"
+NET="${NET}"
+IPV6_METHOD="$IPV6_METHOD"
+
+# Set this only if using "IPV6_METHOD=static"
+#IPV6STATIC="fd00::1234/64"
+
+GATE="${GATE:-none}"
 APT_CACHER_IP="${APT_CACHER_IP:-none}"
-DISABLEIP6=""
-PW='${PW:-none}'
-SSH="${SSH}"
-SSH_AUTHORIZED_KEY="${SSH_AUTHORIZED_KEY}"
-VERBOSE="${VERBOSE}"
-TAGS="${TAGS:-none}"
-VLAN="${VLAN:-none}"
 MTU="${MTU:-1500}"
-GATE="${GATE:-none}"
 SD="${SD:-none}"
-MAC="${MAC:-none}"
 NS="${NS:-none}"
-NET="${NET}"
-FUSE="${ENABLE_FUSE}"
-
+MAC="${MAC:-none}"
+VLAN="${VLAN:-none}"
+SSH="${SSH}"
+SSH_AUTHORIZED_KEY="${SSH_AUTHORIZED_KEY}"
+TAGS="${TAGS:-none}"
+ENABLE_FUSE="$ENABLE_FUSE"
+ENABLE_TUN="$ENABLE_TUN"
 EOF
+fi
```

Comment:
Is this correct?
The file is currently syntactically incorrect.
The is no matching `if` for the `fi` in line 305.
This is currently preventing any installs or updates